.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cinch"]
 	path = cinch
-	url = ../cinch.git
+	url = https://github.com/losalamos/cinch.git
```

Comment:
Where is the CLA and where do I send it after I get Kitware's representative's signature on the form?

As for the cinch submodule change -- this can cause problems if someone isn't cloning directly from https://github.com/losalamos/vpic.git since this is a relative instead of an absolute path. Try the following from the main repo's master branch and then the branch with my changes:
> git clone https://github.com/losalamos/vpic.git
> cd vpic
> git submodule update --init -r
> cd ..
> mkdir vpic2
> cd vpic2
> git clone ../vpic